app/src/main/java/com/beemdevelopment/aegis/util/ViewUtils.java

```diff
@@ -0,0 +1,26 @@
+package com.beemdevelopment.aegis.util;
+
+import androidx.core.graphics.Insets;
+import androidx.core.view.ViewCompat;
+import androidx.core.view.WindowInsetsCompat;
+
+import com.google.android.material.appbar.AppBarLayout;
+
+public class ViewUtils {
```

Comment:
Let's move this to the ``helpers`` package and rename this to ``ViewHelper``.

Reply:
Sure